components/docs-chef-io/content/automate/reusable/md/restore_troubleshooting.md

```diff
@@ -36,22 +38,22 @@ Now check the status of the Automate node and then try running the restore comma
                path = "/bkp/backps"
       ```
 
-   - Update the OpenSearch node with the below template, use the command  `chef-automate config patch os.toml --os`
+      * Update the OpenSearch nodes using the template provided below. Use the command  `chef-automate config patch os.toml --os` to update the Opensearch node configs.
```

Comment:
Replace it with: Update the OpenSearch nodes using the template provided below. Use the `chef-automate config patch os.toml --os` command to update the Opensearch node configs.